frontend/src/components/search/SearchBar.tsx

```diff
@@ -2,15 +2,15 @@ import React from "react";
 
 export default function SearchBar() {
   return (
-    <form className="usa-search usa-search--big" role="search">
+    <div className="usa-search usa-search--big" role="search">
       <label className="usa-sr-only" htmlFor="search-field">
         Search
       </label>
       <input
         className="usa-input maxw-none"
         id="search-field"
         type="search"
-        name="search"
+        name="search-keyword"
```

Comment:
nit: This might not be a keyword. It might be a number, etc… in the future we'll likely even allow more things to be entered here. Perhaps we use something like `search-text-input` or `search-box` (🤷) for both the `id` and `name`?

Reply:
yea let's do `search-text-input` - the name identifies it to the server so we need proper naming here:

Here's an example of what `action.ts` (the server action) sees when we click the search button:
```
  { name: '$ACTION_KEY', value: 'k1960217816' },
    { name: 'search-text-input', value: 'fdsafdsa' },
    { name: 'search-sort-by', value: 'opportunityNumberAsc' }
```

Reply:
I defer to you and @chouinar here for what's most semantic and makes sense for the API. It _could_ be argued that anything entered here is a keyword, even numbers.